tests/sql/common_utils.py

```diff
@@ -0,0 +1,33 @@
+import uuid
```

Comment:
I think it may be more idoomatic for these to live in `conftest.py` then they will be available everywhere in pytest